poolqc/poolingQC_sample.py

```diff
@@ -0,0 +1,300 @@
+import dash_core_components as dcc
+import dash_html_components as html
+import dash.dependencies as dep
+import dash_table as dt
+import pandas
+import plotly.plotly as plotly
+import plotly.graph_objs as go
+import numpy as np
+
+pandas.options.mode.chained_assignment = None
+index = pandas.read_hdf('/home/ijafari/cache_files/rnaseqqc_cache.hd5')
+
+all_runs = index['Sequencer Run Name'].sort_values(ascending=False).unique()
+all_runs = [x for x in all_runs if str(x) != 'nan']
+
+layout = html.Div(children=[
+    dcc.ConfirmDialog(
+        id='error',
+        message='You have input an incorrect URL. Click either "Ok" or "Cancel" to return to the most run.'
```

Comment:
Can this error message be more informative than "incorrect URL"? I believe that no data would be found for the given run if either the run is misspelled or analysis has not yet completed for this run.

Reply:
If the run is misspelled the report will automatically return to the most recent analysis completed.

"The analysis on this run has not been yet completed. Click either 'Ok' or 'Cancel to return to the most recent analysis completed."
Would this be a good alternative?

Reply:
Are you able to distinguish between runs that have not completed analysis (say, those that have only gone through bcl2fastq) and runs that are misspelled?